tests/test_cof_data.py

```diff
@@ -55,4 +56,30 @@ def test_get_cof(mocker, monkeypatch):
     result = get_rounds_for_fund(CommonConfig.COF_FUND_ID)
     assert 2 == len(result[0])
     assert "Round 2 Window 3" == result[0][1]["title"]
-    assert "Round 2 Window 2" == result[0][0]["title"]
\ No newline at end of file
+    assert "Round 2 Window 2" == result[0][0]["title"]
+
+def test_override_fields(monkeypatch):
+    overridable_fields = ["opens"]
+    round_data = {
+        "short_name": "a1b2",
+        "opens": "abc",
+        "dont_override": "original"
+    }
+    monkeypatch.setenv("force_opens_a1b2", "changed")
+    result = override_fields_from_env(round_data, overridable_fields)
+    assert "changed" == result["opens"], "opens field not changed"
+    assert "original" == result["dont_override"], "other field changed"
+
+
+def test_force_open(monkeypatch):
+    monkeypatch.setattr("core.data_operations.round_data.Config.FORCE_OPEN", "True")
+    result = get_round_data("en")
+    assert "2022-01-01 12:00:00" == result[1]["opens"], "opens field not changed"
+    assert "R2W3" == result[1]["short_name"], "other field changed"
+
+def test_override_fields_with_force_open(monkeypatch):
+    monkeypatch.setenv("force_opens_R2W3", "changed")
+    monkeypatch.setattr("core.data_operations.round_data.Config.FORCE_OPEN", "True")
+    result = get_round_data("en")
+    assert "changed" == result[1]["opens"], "opens field not changed"
+    assert "R2W3" == result[1]["short_name"], "other field changed"
```

Comment:
small issue but no newline at end of file so some unix tools might bug out

Reply:
Also, some pytest maintainers recommend using default assertions (i.e `assert x == y`) - as pytest does some magic to print pretty errors for you if you don't provide one

Reply:
Thanks - didn't have pre-commit installed for this repo.